scripts/azure-templates-stages.yml

```diff
@@ -472,21 +472,6 @@ stages:
             postBuildSteps:
               - pwsh: Remove-Item ./output/native/ -Recurse -Force -ErrorAction Continue
                 displayName: Delete the native folder
-        - template: azure-templates-bootstrapper.yml # Build Managed (Linux)
-          parameters:
-            name: managed_linux
-            displayName: Managed (Linux)
```

Comment:
Is this step failing? Maybe we should keep it for people building on linux? At least we can make sure we do not break anything?

Reply:
Yes, let's keep it, and the work you've done to include netcore support will allow for bringing it back. Thanks!